setup.cake

```diff
@@ -1,4 +1,4 @@
-#load nuget:https://www.myget.org/F/cake-contrib/api/v2?package=Cake.Recipe&prerelease
+#load nuget:?package=Cake.Recipe&prerelease
```

Comment:
@agc93 is there a reason that you have moved to the released version on NuGet?  Best advice at the minute, until 0.3.0 of Cake.Recipe is released, is to use the version from the MyGet feed.

Reply:
Ah, I'd assumed the NuGet feed was more of a "stable" version. I'll change that back.

Reply:
Unfortunately, not.  There has been a lot of `churn` lately, and the best version to use is the version from MyGet, as it has the updated pinned version of addins/tools that will work with Cake 0.25.0.  Once the PR that you linked to is complete, I will likely release 0.3.0 of Cake.Recipe to NuGet, which will work with 0.28.1 of Cake.